gr-qtgui/python/qtgui/bindings/video_display_python.cc

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+/***********************************************************************************/
+/* This file is automatically generated using bindtool and can be manually edited  */
+/* The following lines can be configured to regenerate this file during cmake      */
+/* If manual edits are made, the following tags should be modified accordingly.    */
+/* BINDTOOL_GEN_AUTOMATIC(0)                                                       */
+/* BINDTOOL_USE_PYGCCXML(0)                                                        */
+/* BINDTOOL_HEADER_FILE(video_display.h)                                        */
```

Comment:
Just a nit pick, but please fix whitespace here :wink:

Reply:
got it

Reply:
@luzpaz A few spelling/whitespace fixes can be useful, but improvements in the content of documentation would be even more appreciated. And, of course, features, technical improvents, etc.